.circleci/config.yml

```diff
@@ -538,3 +583,19 @@ workflows:
             branches:
               only:
                 - master
+      - pytorch_windows_build_worker_0:
+          name: win_test_worker_0
+      - pytorch_windows_build_worker_1:
+          name: win_test_worker_1
+      - pytorch_windows_build_worker_2:
+          name: win_test_worker_2
+      - pytorch_windows_build_worker_3:
+          name: win_test_worker_3
+      - pytorch_windows_build_worker_4:
+          name: win_test_worker_4
+      - pytorch_windows_build_worker_5:
+          name: win_test_worker_5
+      - pytorch_windows_build_worker_6:
+          name: win_test_worker_6
+      - pytorch_windows_build_worker_7:
+          name: win_test_worker_7
```

Comment:
This job seems extremely expensive to run basically the way the math works this would be `500 credits x 55 minutes * 8 workers`

We should at the very least gate these to run only on master and even, I think, gate these to only run when someone manually approves these to run.

cc @malfet, @yns88

Reply:
Does every worker need a GPU machine to run? Can some of the work be shifted to CPU-only machines? Can the amount of coverage on the Windows build be reduced so it doesn't need 8 workers?

If not, then yes, we'll need to limit this to master-only (that might be sufficient, since this repository isn't nearly as active as pytorch/pytorch).

Reply:
Considering the parallel jobs (8 jobs with most 55minutes for one) seems not much more efficient than one job (1 hour 36 minutes). I have changed the parallel jobs to a single one with manually approval.